SCClassLibrary/Common/Control/Server.sc

```diff
@@ -351,13 +353,13 @@ Server {
 
 	}
 
+	numClients { ^numClients ? options.maxLogins }
```

Comment:
FWIW: In my testing, `a ? b` is no more and no less than `a ?? { b }` (sometimes one is faster, sometimes the other is, randomly). But if a is not nil, `a ? b.method` is *always* 20-25% slower than `a ?? { b.method }`. Let's not overextend that one surprising finding from awhile ago. In the present case here, `??` is better.

Reply:
ok!